packages/docusaurus-plugin-content-docs/src/globalData.ts

```diff
@@ -20,13 +24,44 @@ export function toGlobalDataDoc(doc: DocMetadata): GlobalDoc {
   };
 }
 
+export function toGlobalSidebar(
+  sidebarId: string,
+  sidebar: Sidebar,
+  version: LoadedVersion,
+): GlobalSidebar {
+  const sidebarFirstDocId = getFirstDocIdOfSidebar(sidebar);
+  let sidebarFirstDocPermalink;
```

Comment:
Typically we prefer ternary operators for this. It won't compromise much readability but it enables us to use `const` and stricter types.